vtex-apps/my-account
# Fix receiver name creation problem

#### What did you change? \*

- Passing the receiver name while creating an address.

#### Why? \*

- The receiver name is mandatory to finish a purchase

#### How to test it? \*

- Create an address: 

https://storev1--recorrenciaqa.myvtex.com/account#/addresses

- Check if the receiver name is present afterwards...

#### Related to / Depends on?



#### Types of changes \*
